src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -404,13 +425,7 @@ def hasNodeGenData(self, key: str) -> bool:
 
     def getNode(
         self, key: str
-    ) -> Union[
-        ParameterConfig,
-        GenKwConfig,
-        GenDataConfig,
-        SummaryConfig,
-        ExtParamConfig,
-    ]:
+    ) -> Union[ParameterConfig, GenDataConfig, SummaryConfig, ExtParamConfig,]:
```

Comment:
Can this be further simplified by removing `ExtParamConfig`?

Reply:
yes